src/java.base/share/classes/javax/security/auth/PrivateCredentialPermission.java

```diff
@@ -127,7 +127,7 @@ public final class PrivateCredentialPermission extends Permission {
     /**
      * @serial
      */
-    private boolean testing = false;
+    private final boolean testing = false;
```

Comment:
This should really use java.security.debug so this tracing can be enabled at runtime, but that's probably more of a separate fix - feel free to file a separate issue.

Reply:
I'll file a bug later today and add to this bug report.

Reply:
You must mean sun.security.util.Debug.

Reply:
JDK-8291974